mage_ai/orchestration/pipeline_scheduler.py

```diff
@@ -269,39 +272,35 @@ def queued_block_runs(self) -> List[BlockRun]:
                     block.all_upstream_blocks_completed(completed_block_uuids)
 
             if completed:
-                block_run.update(status=BlockRun.BlockRunStatus.QUEUED)
                 queued_block_runs.append(block_run)
 
         return queued_block_runs
 
     def __schedule_blocks(self, block_runs: List[BlockRun] = None) -> None:
         # TODO: implement queueing logic
         block_runs_to_schedule = self.queued_block_runs if block_runs is None else block_runs
-        block_runs_to_schedule = self.__fetch_crashed_block_runs() + block_runs_to_schedule
+        # block_runs_to_schedule = self.__fetch_crashed_block_runs() + block_runs_to_schedule
 
         for b in block_runs_to_schedule:
             tags = dict(
                 block_run_id=b.id,
                 block_uuid=b.block_uuid,
             )
-
             b.update(
-                started_at=datetime.now(),
-                status=BlockRun.BlockRunStatus.RUNNING,
+                status=BlockRun.BlockRunStatus.QUEUED,
             )
 
             self.logger.info(
                 f'Start a process for BlockRun {b.id}',
                 **self.__build_tags(**tags),
             )
-            proc = create_process(run_block, (
+            job_uuid = worker_manager.add_job(run_block, (
                 self.pipeline_run.id,
                 b.id,
                 get_variables(self.pipeline_run),
                 self.__build_tags(**tags),
             ))
-            execution_process_manager.set_block_process(self.pipeline_run.id, b.id, proc)
-            proc.start()
+            execution_job_manager.set_block_job(self.pipeline_run.id, b.id, job_uuid)
```

Comment:
You'll need to update the places that calls `execution_process_manager.has_block_process`